QueueProcessors/AutoreductionProcessor/post_process_admin.py

```diff
@@ -27,6 +27,10 @@
 import traceback
 from contextlib import contextmanager
 
+
+# Ensure that Mantid does not attempt to plot to display
+os.environ['MPLBACKEND'] = 'Agg'
```

Comment:
You should be able to remove this from here now as this is handled in the docker file